tests/framework/utils/k8s_helper.go

```diff
@@ -429,23 +425,29 @@ func (k8sh *K8sHelper) IsPodWithLabelPresent(label string, namespace string) boo
 	return count > 0
 }
 
-//WaitForLabeledPodToRun returns true if a Pod is running status or goes to Running status within 90s else returns false
-func (k8sh *K8sHelper) WaitForLabeledPodToRun(label string, namespace string) error {
+//WaitForLabeledPodsToRun returns true if a Pod is running status or goes to Running status within 90s else returns false
```

Comment:
Could you add a space here at the comment begin `// WaitForLa...`

Reply:
i'd like to hold off on whitespace changes and get this merged. It was pre-existing in the code. Totally agreed on the whitespace when i touch this in the future.